modules/backing-up-red-hat-quay-standalone.adoc

```diff
@@ -0,0 +1,108 @@
+:_content-type: PROCEDURE
+[[backing-up-red-hat-quay-standalone]]
+= Backing up {productname} on standalone deployments 
+
+This procedure describes how to create a backup of {productname} on standalone deployments. 
+
+.Prerequisites
+
+.Procedure 
+
+. Create a temporary backup directory, for example, `quay-backup`: 
++
+[source,terminal]
+----
+$ mkdir /tmp/quay-backup
+----
+
+. Change into a directory that bind-mounts to `/conf/stack` inside of the container by running the following command: 
++
+[source,terminal]
+----
+$ cd /opt/quay
+----
+
+. Compress the contents of your {productname} deployment into an archive in the `quay-backup` directory by entering the following command: 
++
+[source,terminal]
+----
+$ tar cvf /tmp/quay-backup/quay-backup.tar.gz *
+----
++
+Example output:
++
+[source,terminal]
+----
+config.yaml
+config.yaml.bak
+extra_ca_certs/
+extra_ca_certs/ca.crt
+ssl.cert
+ssl.key
+----
+
+. Redirect the contents of your `conf/stack/config.yaml` file to your temporary `quay-config.yaml` file by entering the following command: 
++
+[source,terminal]
+----
+$ podman exec -it quay cat /conf/stack/config.yaml > /tmp/quay-backup/quay-config.yaml 
+----
+
+. Obtain the `DB_URI` located in your temporary `quay-config.yaml` by entering the following command:
++
+[source,terminal]
+----
+$ grep DB_URI /tmp/quay-backup/quay-config.yaml
+----
++
+Example output:
++
+----
+$ postgresql://<username>:test123@172.24.10.50/quay
+----
+
+. Extract the PostgreSQL contents to your temporary backup directory in a backup .sql file by entering the following command:
++
+[source,terminal]
+----
+$ pg_dump -h 172.24.10.50 -d quay -U <username> -W -O > /tmp/quay-backup/quay-backup.sql
+----
+
+. Print the contents of your `DISTRIBUTED_STORAGE_CONFIG` by entering the following command: 
++
+[source,yaml]
+----
+DISTRIBUTED_STORAGE_CONFIG:
+   default:
+       - RadosGWStorage
+       - access_key: <access_key>
+         bucket_name: quay
+         hostname: 172.24.10.50
+         is_secure: false
+         port: 9000
+         secret_key: <secret_key>
+         storage_path: /datastorage/registry
+DISTRIBUTED_STORAGE_DEFAULT_LOCATIONS: []
+----
+
+. Export the `AWS_ACCESS_KEY_ID` by using the `access_key` credential obtained in Step 7:
++
+[source,terminal]
+----
+$ export AWS_ACCESS_KEY_ID=<access_key> 
+----
+
+. Export the `AWS_SECRET_ACCESS_KEY` by using the `secret_key` obtained in Step 7:
++
+[source,terminal]
+----
+$ export AWS_SECRET_ACCESS_KEY=<secret_key>
+----
+
+. Sync the `quay` bucket to the `/tmp/quay-backup/blob-backup/` directory from the `hostname` of your `DISTRIBUTED_STORAGE_CONFIG`:
++
+[source,terminal]
+----
+$ aws s3 sync --no-verify-ssl --endpoint-url http://172.24.10.50:9000 s3://quay/ /tmp/quay-backup/blob-backup/
+----
+
```

Comment:
We should probably recommend to delete the `quay-config.yaml` file as it contains sensitive information and the configuration is already backed up in the `quay-backup.tar.gz` file.